### ISSUE TITLE
[docs] section on typescript

### ISSUE THREAD

**Issue by ghost** [lit/lit-element#420]
Create a section on TypeScript

- decorators
- build issue

**Comment by katejeffreys** [comment]
https://github.com/Polymer/lit-element/issues/410

**Comment by katejeffreys** [comment]
https://github.com/Polymer/lit-element/issues/211

**Comment by mcalthrop** [comment]
@ghost What do we have to do in order to make the decorator (ie, Typescript) examples actually work?

Currently, using the decorator code examples (eg [Use LitElement TypeScript decorators](https://lit-element.polymer-project.org/guide/start#use-litelement-typescript-decorators)) produces the error described in #694, which makes the example useless.

**Comment by LarsDenBakker** [comment]
You need to compile the typescript to javascript

**Comment by mcalthrop** [comment]
@LarsDenBakker Yep, thanks, I get that. However I think providing examples that don't actually work out of the box is not useful.

My ideal solution is where _all_ files and config required are listed.

**Comment by mcalthrop** [comment]
@LarsDenBakker Just saw your comment update... I still think the very minimum the docs should do is mention the fact that you need to do more than what is in the docs, and link to somewhere useful that shows what needs to be done.

**Comment by mcalthrop** [comment]
What's also odd is that I downloaded the stackblitz file, and tried to run it locally – no joy.

Here's what I did:
- clicked on the **Download Project** icon
- unzipped the downloaded file
- in the terminal, did the following:
  - `npm install`
  - `npm install --save-dev polymer-cli`
  - `npx polymer serve`
  - opened the browser at http://127.0.0.1:8000/components/1r5zv2.run/
And the page was empty.

When inspecting the `index.html` file, this kind of makes sense, because it doesn't load either the `index.ts` file or the `my-element.ts` file (or their transpiled JS versions)... so tbh I don't know how it actually runs in the stackblitz iframe in the lit-element docs.

Software versions:

```sh
$ node --version
v10.15.3
$ npm --version
6.4.1
```
